pymongo/pool.py

```diff
@@ -407,6 +411,32 @@ def _cond_wait(condition: threading.Condition, deadline: Optional[float]) -> boo
     return condition.wait(timeout)
 
 
+def _get_timeout_details(options: PoolOptions) -> dict[str, float]:
+    details = {}
+    timeout = _csot.get_timeout()
+    socket_timeout = options.socket_timeout
+    connect_timeout = options.connect_timeout
+    if timeout:
+        details["timeout"] = timeout * 1000
```

Comment:
I suspect "timeout" will be confusing to users. Perhaps "operation timeout"? Also if _csot.get_timeout is enabled then should we avoid adding socketTimeoutMS here since CSOT overrides socketTimeoutMS?

Reply:
I went with `operationTimeoutMS`. Agreed, if CSOT is present then socketTimeoutMS should not be displayed.